src/cascadia/TerminalApp/defaults.json

```diff
@@ -0,0 +1,216 @@
+// THIS IS AN AUTO-GENERATED FILE! Changes to this file will be ignored.
+{
+    "globals":
+    {
+        "alwaysShowTabs": true,
+        "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",

```

Comment:
ah, good question: how do we change the default on first load anyway? to pwsh core?

Reply:
This is going to be something I'm figuring out in the dynamic profiles PR. When we're deciding to write the user settings template, I'll need to figure out what to do there